client/fingerprint/env_aws_cpuinfo.go

```diff
@@ -0,0 +1,1620 @@
+// Code generated from hashicorpnomad/ec2info; DO NOT EDIT.
+
+package fingerprint
+
+type ec2specs struct {
+	mhz   float64
+	cores int
+}
+
+func specs(ghz float64, cores int) ec2specs {
+	return ec2specs{
+		mhz:   ghz * 1000,
+		cores: cores,
+	}
+}
+
+var ec2ProcSpeedTable = map[string]map[string]ec2specs{
+
+	"ap-northeast-1": {
```

Comment:
Are we worried about specs for the same instance varying by region?  I'd find that very surprising if that's the case?

Looking at the current file, I don't see any divergence.  Deduplicating it reduces the elements from 1569 map entries to 212.

Also, I suspect some missing data: `c5a.16xlarge` and `c5.18xlarge` are available in multiple regions according to their press releases but they are only listed for eu-north-1 here: https://aws.amazon.com/blogs/aws/new-amazon-ec2-c5a-instances-powered-by-2nd-gen-amd-epyc-processors/, and https://aws.amazon.com/blogs/aws/now-available-compute-intensive-c5-instances-for-amazon-ec2/ -